spec/fixtures/files/sac_imports_src/NAV3_fixture.csv

```diff
@@ -1,39 +1,39 @@
-"4200006",1,"QUAL-1",2019-07-13,2025-12-31,2026-01-01,2029-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200007",1,"QUAL-1",2022-01-23,2028-12-31,2029-01-01,2032-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200008",1,"QUAL-1",2018-07-13,2024-12-31,2025-01-01,2028-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200009",1,"QUAL-1",2023-01-22,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200010",1,"QUAL-2",1982-03-30,2010-03-02,"NULL","NULL","Leiter*in Kinderbergsteigen"
-"4200000",1,"QUAL-3",2023-11-11,"NULL","NULL","NULL","SAC Tourenleiter - Aspirant*in"
-"4200001",1,"QUAL-1",2018-03-10,2024-12-31,2025-01-01,2028-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200002",1,"QUAL-1",2018-08-04,2024-12-31,2025-01-01,2028-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200003",1,"QUAL-1",2018-07-07,2024-12-31,2025-01-01,2028-12-31,"SAC Tourenleiter*in Sportklettern"
-"4200004",1,"QUAL-3",2020-03-10,"NULL","NULL","NULL","Wanderleiter*in bis T4"
-"4200005",1,"QUAL-3",2018-04-11,"NULL","NULL","NULL","Wanderleiter*in Kantonal + SBV"
-"4200006",1,"QUAL-3",2018-06-05,"NULL","NULL","NULL","Schneeschuhleiter*in bis WT4"
-"4200007",1,"QUAL-3",2018-06-05,"NULL","NULL","NULL","Wanderleiter*in bis T4"
-"4200008",1,"QUAL-1",2018-04-21,2025-12-31,2026-01-01,2029-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200009",1,"QUAL-2",2020-10-10,"NULL","NULL","NULL","SAC Tourenleiter*in Bergwandern"
-"4200010",1,"QUAL-3",2022-06-10,"NULL","NULL","NULL","SAC Tourenleiter - Aspirant*in"
-"4200000",1,"QUAL-2",2017-09-10,"NULL","NULL","NULL","SAC Tourenleiter*in Bergwandern"
-"4200001",1,"QUAL-1",2022-01-22,2028-12-31,2029-01-01,2032-12-31,"SAC Tourenleiter*in 1 Winter Senioren"
-"4200002",1,"QUAL-1",2019-07-19,2028-12-31,2029-01-01,2032-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200003",1,"QUAL-3",2021-01-29,"NULL","NULL","NULL","Wanderleiter*in bis T4"
-"4200004",1,"QUAL-1",2023-03-11,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200005",1,"QUAL-1",2023-06-25,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200006",1,"QUAL-1",2024-03-10,2030-12-31,2031-01-01,2034-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200007",1,"QUAL-1",2023-06-25,2030-12-31,2031-01-01,2034-12-31,"Murmeltierdoktor*in"
-"4200008",1,"QUAL-1",2022-03-12,2028-12-31,2029-01-01,2032-12-31,"SAC Tourenleiter*in 1 Winter"
-"4200009",1,"QUAL-2",2017-10-14,"NULL","NULL","NULL","SAC Tourenleiter*in Bergwandern"
-"4200010",1,"QUAL-1",2023-06-23,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in Alpinwandern"
-"4200000",1,"QUAL-2",2021-06-12,"NULL","NULL","NULL","SAC Tourenleiter*in Bergwandern"
-"4200001",1,"QUAL-1",2021-06-06,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in Sportklettern"
-"4200002",1,"QUAL-1",2022-07-30,2029-12-31,2030-01-01,2033-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200003",1,"QUAL-1",2020-07-03,2028-12-31,2029-01-01,2032-12-31,"SAC Tourenleiter*in 1 Sommer"
-"4200004",1,"QUAL-1",2024-02-04,2030-12-31,2031-01-01,2034-12-31,"SAC Tourenleiter*in 1 Winter"
```

Comment:
I'm not sure about the number and data normalizations. How are these in the original data?

Reply:
They are not quoted.

Reply:
I'm unsure if dates should be quotes?

Reply:
Damn in the WSO2 source they are...

Reply:
I'll only normalize WSO21 for now, there it survives the roundtrip.